crates/xxi-node/src/commons/message.rs

```diff
@@ -80,7 +80,11 @@ pub enum OrderbookRequest {
         os: Option<String>,
         signature: Signature,
     },
+    // TODO(holzeis): This orderbook request seems to be unused and a duplicate to our http
+    // endpoint. I guess we can remove it.
     InsertOrder(NewLimitOrder),
+    // TODO(holzeis): This orderbook request seems to be unused and a duplicate to our http
+    // endpoint. I guess we can remove it.
```

Comment:
Nope, this is how the maker communicates with the orderbook. We want to get rid of the HTTP requests instead and use a WS connection between app and coordinator.

Reply:
Makes sense, the dev-maker is using the http endpoint though. I guess that confused me 🙂

Reply:
Fair enough. I was too lazy to update the dev maker back when I changed this 😔